frontend/packages/dev-console/src/components/import/import-validation-utils.ts

```diff
@@ -30,8 +31,11 @@ export const validationSchema = yup.object().shape({
       .required('Required'),
   }),
   application: yup.object().shape({
-    name: yup.string(),
-    selectedKey: yup.string(),
+    selectedKey: yup.string().required('Required'),
+    name: yup.string().when('selectedKey', {
+      is: (selectedKey) => selectedKey !== UNASSIGNED_KEY,
```

Comment:
Is the default for selectedKey `UNASSIGNED_KEY`?

Reply:
No, this is to set the name to not be required when the selectedKey is `UNASSIGNED_KEY`. The default is either an existing application or `CREATE_APPLICATION_KEY`